src/components/PostPageDetail/index.js

```diff
@@ -740,7 +740,7 @@ const PostPageDetailIdComponent = (props) => {
             }}
           />
 
-          <BlockComponent ref={refBlockComponent} refresh={updateFeed} screen="post_detail_page" />
+          <BlockComponent ref={refBlockComponent} screen="post_detail_page" />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `refresh` prop from the `BlockComponent` might cause issues if the `BlockComponent` relies on it for some functionality. If the `updateFeed` function is no longer needed, please make sure that its absence doesn't cause any unintended side effects.
```diff
-    <BlockComponent ref={refBlockComponent} refresh={updateFeed} screen="post_detail_page" />
+    <BlockComponent ref={refBlockComponent} screen="post_detail_page" />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->